Source/Scene/Model.js

```diff
@@ -1465,11 +1650,19 @@ define([
         var programs = model.gltf.programs;
         var shaders = model._loadResources.shaders;
         var program = programs[id];
+        var extensionsUsed = model.gltf.extensionsUsed;
 
         var attributeLocations = createAttributeLocations(program.attributes);
         var vs = getShaderSource(model, shaders[program.vertexShader]);
         var fs = getShaderSource(model, shaders[program.fragmentShader]);
 
+        // Check for extensions
+        if (defined(extensionsUsed) && extensionsUsed.constructor === Array) {
```

Comment:
Is this check needed: `extensionsUsed.constructor === Array`

`extensionsUsed` is defined to be an array.